### DISCUSSION TITLE
Failed to access data from shortcut

### DISCUSSION THREAD

**Discussion by kisonho** [utmapp/UTM#3774] [UTM Help (macOS)] [answered]
When importing *.utm file created from another user and saved to a shared folder (e.g. external disks), there is an error message saying "_Failed to access data from shortcut._". 

Other information is listed below:
- System information: MacOS 12.2.1, both x86 and ARM architecture.
- Virtual machine information: Windows 11 (x86/ARM) with 8GB RAM.

Any solutions?

**Comment by runwaygoats** [comment, marked as answer]
Delete view.plist in Virtual Machine folder.

**Reply by kisonho** [reply]
That works! Thanks!

**Reply by Penffy** [reply]
how to open virtual machine folder

**Reply by runwaygoats** [reply]
> how to open virtual machine folder

Right Click on Virtual Machine and select "Show in Finder"
Then Right Click again and "Show Package Contents"
.utm file is actually a folder, even if Finder shows it as a file (or app).

**Reply by Penffy** [reply]
Thanks !!!

**Reply by ianlewisau** [reply]
Yes, it works - thanks!  The remaining question for me is why it failed to load? I've got a windows XP VM to run an old app and it only gets used very occasionally, but it seems that every time I launch it there is some other problem. I've rebuilt the VM from scratch a couple of times and was really happy to find this fix.

**Reply by Scc33** [reply]
What if you don't have a view.plist?

**Reply by EricNavarroLosier** [reply]
> > how to open virtual machine folder
> 
> Right Click on Virtual Machine and select "Show in Finder" Then Right Click again and "Show Package Contents" .utm file is actually a folder, even if Finder shows it as a file (or app).

Thank you so much!

**Reply by dedyks** [reply]
Works like a charm

**Reply by helinwang** [reply]
Thanks it works. I have to delete the file, close UTM (don't open the VM yet), open UTM and then open VM.

**Comment by alxiong** [comment]
I had run into a similar problem, but I don't have `view.plist` in the `./Virtual Machine` folder. 

What I tried but doesn't work is: re-download from the [gallery](https://mac.getutm.app/gallery/), unzip it, then open/import to UTM.

What ends up working for me is: directly click on `Open in UTM` option in the gallery.
(not sure why it works but the other doesn't, but at least it works for me now.)

My Env: MacOS M1 chip, running the latest UTM and [Debian 10 minimal](https://mac.getutm.app/gallery/debian-10-4-minimal).
Forgot to mention, the problem appear when I delete a previously working Debian UTM file, and try to re-import a new one.

**Reply by EricNavarroLosier** [reply]
Have you tried what @runwaygoats explained about "Show Package Contents"? 
That works like a charm!
You just right-click on the .utm file that contains the VM.

![Screenshot 2022-09-29 at 17 23 49](https://user-images.githubusercontent.com/4154035/193073021-bdcf0679-a9db-4ad8-bd44-215f079f223a.png)

**Reply by Bestshotuk** [reply]
I had a similar issue when I restored my M1 Mac from a Time Machine backup.
Failed to access data from shortcut.
I simply created a clone of that shortcut and my windows vm booted from that cloned shortcut without issues.

**Comment by d0wntime** [comment]
This is still happening, even when there's no view.plist inside the bundle.

**Reply by d0wntime** [reply]
even with a freshly-imported VM.

**Reply by d0wntime** [reply]
The problem turns out to be that bookmarks are now also being stored for stale/removed VMs in com.utmapp.UTM.plist in ~/Library/Containers/com.utmapp.UTM/Data/Library/Preferences.  Manual editing of this file to remove the bookmarks is required.  This seems kind of onerous, should be marked a bug unless I'm missing something.

**Reply by d0wntime** [reply]
and, better still, manual editing of that file to remove a bookmark while the app is quit seems to invalidate the file on re-starting the application.  all VMs need to be re-added.  this is a bug.

**Reply by ghost** [reply]
Thanks, your hint solved my issue.
### My way of solution
#### Reveal hidden files
```
defaults write com.apple.Finder AppleShowAllFiles true
killall Finder
```
#### Quit UTM and delete `.com.apple.containermanagerd.metadata` in:
```
/Users/username/Library/Containers/com.utmapp.UTM
```
#### Reverse "Reveal hidden files"
```
defaults write com.apple.Finder AppleShowAllFiles false
killall Finder
```

**Reply by graceannx** [reply]
Thank you so much!
